src/gui/CMakeLists.txt

```diff
@@ -118,6 +118,10 @@ transferlistwidget.cpp
 updownratiodlg.cpp
 )
 
+if (APPLE)
+    list(APPEND QBT_GUI_SOURCES macutilities.mm)
+endif (APPLE)
```

Comment:
I think it is better to add `macutilities.h` to QBT_GUI_HEADERS as well, since the code below does it at least.